example/docker-compose.yaml

```diff
@@ -2,7 +2,7 @@ version: "3.4"
 
 services:
   dmss:
-    image: datamodelingtool.azurecr.io/dmss:latest
+    image: datamodelingtool.azurecr.io/dmss:dev
```

Comment:
no tags dev in the registry. 
Either use latest or one of the versions ( 1.0.0 etc)

Reply:
nice catch, it should be latest, I have reverted it